.gitignore

```diff
@@ -59,3 +59,10 @@ ruby
 /node_modules
 yarn-debug.log*
 .yarn-integrity
+
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
```

Comment:
Tiny nitpick: There are a few duplicate lines in `.gitignore` from running the webpack install process again (once in the past for 3.x, once now for 4.x).

The new auto-generated entries in the `.gitignore` are good. We can delete the old ones (just above).

(I notice the difference is: the new ones include `/yarn-error.log` to be gitignored, which the old ones don't.)